### PR TITLE
release(go-sdk): v0.2.2 w/ the OpenFgaClient & BatchCheck

### PR DESCRIPTION
**Pull request by rhamzeh**
## References

- Generates: https://github.com/openfga/go-sdk/pull/14
- Related: https://github.com/openfga/sdk-generator/issues/23

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: config/clients/go/template/configuration.mustache [hunk 1]
```diff
@@ -18,12 +18,12 @@ type RetryParams struct {
 
 // Configuration stores the configuration of the API client
 type Configuration struct {
-    ApiScheme      string                   `json:"apiScheme,omitempty"`
-    ApiHost        string                   `json:"apiHost,omitempty"`
-    StoreId        string                   `json:"storeId,omitempty"`
-    Credentials    *credentials.Credentials `json:"credentials,omitempty"`
-    DefaultHeaders map[string]string        `json:"defaultHeader,omitempty"`
-    UserAgent      string                   `json:"userAgent,omitempty"`
+	ApiScheme      string                   `json:"api_scheme,omitempty"`
```
Comment (adriantam) [inline config/clients/go/template/configuration.mustache:21]: Is there any reason we change the spacing?

File: config/common/files/.github/PULL_REQUEST_TEMPLATE.md.mustache [hunk 2]
```diff
@@ -1 +0,0 @@
-We are not accepting new Pull Requests for the time being. Please raise an issue or contact us in the [support community]({{supportInfo}}) instead.
```
Comment (adriantam) [inline config/common/files/.github/PULL_REQUEST_TEMPLATE.md.mustache:1]: Any reason we remove these?

**Review by adriantam — APPROVED** [review]

File: config/common/files/.github/PULL_REQUEST_TEMPLATE.md.mustache (hunk 2 above)
Comment (rhamzeh) [reply config/common/files/.github/PULL_REQUEST_TEMPLATE.md.mustache:1]: To encourage people to send PRs, we'll do the heavy work of porting it over to the generator. Asking them to submit PRs here is not always easy for them

File: config/clients/go/template/configuration.mustache (hunk 1 above)
Comment (rhamzeh) [reply config/clients/go/template/configuration.mustache:21]: Don't worry about the spacing, go fmt handles fixing it after generation (so in the generated code, the spacing will be correct).

In this case the updated spacing is "correct", the older spacing was being formatted away

**Review by adriantam — APPROVED** [review]

**Review by adriantam — APPROVED** [review]

**Pull request merged**